test/widget_test.dart

```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```

Comment:
Please update this test to match the application feature, see existing example that is simple enough (it types a latter inside the game panel and then ensure that letter has been consumed)
Once updated, try to run it against the Linux build and add the recording to the PR!

I know this is time consuming, but we are all learning from this and this will increase overall quality!

Reply:
Hey, I just saw that this test was created, I didn't really mean to do so.

Could you elaborate what you were looking for? Sorry but I just don't really understand.
As the test was added by accident, do you just want me to delete it?

Reply:
Oh got it... this seems to be the default counter app Flutter has inside it's template project I guess...
- then just delete it!